.github/CONTRIBUTING.md

```diff
@@ -21,9 +21,6 @@ time as the team learns, listens and refines how we work with the community.
   * [Suggesting Enhancements](#suggesting-enhancements)
   * [Help Wanted](#help-wanted)
 
-[Additional Notes](#additional-notes)
-  * [Issue and Pull Request Labels](#issue-and-pull-request-labels)
```

Comment:
Do we want a link to `labels.md` from anywhere? maybe the root README?

Reply:
I'm 👍 to just linking to the new doc here for now

Reply:
6f61a65 has a section called Process Docs that lists some interesting things. We can refine/expand this later.